rpg-sheet/src/components/AttributeList/AttributeList.js

```diff
@@ -0,0 +1,14 @@
+import { Attribute } from "../Attribute/Attribute"
+
+export const AttributeList = ({attributes}) => {
+  const attributesEntries = Object.entries(attributes)
```

Comment:
Maybe you should use destructuring to get the attributes values.
Once we know the attributes object is something like this: 

`attributes: {
      strength: 50,
      wisdom: 50, 
      intelligency: 50, 
      agility: 50, 
      charisma: 50, 
      dexterity: 50
    }, `

Destructuring can be used like this:
`const {strength,dexterity,agility,intelligency,wisdom,charisma} = attributes`

You can read more about in this article:
(https://medium.com/@lcriswell/destructuring-props-in-react-b1c295005ce0)

Reply:
The idea here is that I want to get the value and the key, that's why I didn't use destructuring. I thought maybe if the attributes changed, the keys would change also, so I wanted to make it dinamic